src/components/content/order/orderStatus/OrderSubmitResult.tsx

```diff
@@ -37,6 +78,19 @@ export const OrderSubmitResult = ({
                 type={type}
                 action={
                     <>
+                        {type === 'error' ? (
```

Comment:
the information here is a bit misplaced. better to place everything one below the other -> retry request, order contact service vendor, order duration.